.eslintrc.json

```diff
@@ -85,7 +85,7 @@
       "last"
     ],
     "complexity": [
-      1,
+      0,
```

Comment:
Could we maybe keep these changes in their own PRs in the future? Since this is just a warning, it's not something that blocks these changes or needs to go into the release branch. No need to change it now though

Reply:
Saves having to do a separate PR and is harmless.